src/sections/dataset/dataset-files/dataset-upload-files-button/DatasetUploadFilesButton.tsx

```diff
@@ -16,11 +17,18 @@ export function DatasetUploadFilesButton() {
   const { user } = useSession()
   const { dataset } = useDataset()
   const navigate = useNavigate()
+  const { datasetFileStore } = useGetDatasetFileStore(dataset?.id)
 
   if (!user || !dataset?.permissions.canUpdateDataset) {
     return <></>
   }
 
+  // This is a temporary solution as this use case doesn't exist in js-dataverse yet and the API should also return the file store type rather than name only.
```

Comment:
Can we create an issue for this and update the comment with a TODO pointing to the issue?